src/screens/DomainScreen/elements/RenderItem.js

```diff
@@ -77,8 +60,8 @@ const RenderItem = ({
         }
 
         if (item.latest_reactions.downvotes !== undefined) {
-          let downvotes = item.latest_reactions.downvotes.filter(
-            (vote) => vote.user_id === selfUserId,
+          const downvotes = item.latest_reactions.downvotes.filter(
+            (vote) => vote.user_id === selfUserId
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Same issue as above, you should check if the array is empty instead of checking for `undefined`.
```diff
- if (downvotes !== undefined) {
+ if (downvotes.length > 0) {
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->